src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -0,0 +1,365 @@
+  <figcaption class="w-figcaption">
+    Amstelvar's "n" contours in Italic (12 point, regular weight, normal width),
+    and in Roman. Image supplied by David Berlow, type designer and typographer
+    at Font Bureau.
+  </figcaption>
+</figure>
+
+After the switch to Italic, the axes available to the user should be the same
+as those for the Roman, just as the character set should be the same.
+
+A glyph substitution capability can also be seen for individual glyphs, and used
+anywhere in the design space of a variable font. For example, a dollar sign
+design with two vertical bars works best at larger point sizes, but at smaller
+point sizes a design with only one bar is better. When we have fewer pixels
+for rendering the glyph, a two bar design can become illegible. To combat this,
+much like the Italic axis, a glyph substitution of one glyph for another can
+occur along the **Optical Size** axis at a point decided by the type designer.
+
+In summary, where the contours allow for it, type designers can create fonts
+that interpolate between various masters in a multi-dimensional design space.
+This gives you granular control over your typography, and a great deal of power.
+
+## Axes Definitions
+
+Since the font developers define which axes are available in their fonts, it is
+essential to check the font's documentation to know what is available. For
+example, in the Gingham variable font designed by Christoph Koeberlin, there are
+two axes available, Width and Weight. The Amstelvar variable font does not
+contain a Slant axis, but does have an axis it calls Grade, plus many more axes.
+
+A Grade axis is interesting as it changes the weight of the font without
+changing the widths, so line breaks do not change. By playing with a
+Grade axis, you can avoid being forced to fiddle with changes to the Weight axis
+that effects the overall width, and then changes to the Width axis that effect
+the overall weight. This is possible because the Amstelvar default style has
+been deconstructed in the 4 fundamental aspects of form: black or positive
+shapes, white or negative shapes, and the X and Y dimensions. These 4 aspects
+can be mixed to form the other styles, such as Width and Weight, in the way
+that primary colors can be mixed to create any other color.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/amstel-grade-e.mp4" 
+            type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Grade axis being changed on the fly.
+  </figcaption>
+</figure>
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/variable-font-experiments?previewSize=100&attributionHidden=true"
+    alt="Variable fonts experiments on Glitch."
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
```

Comment:
Embedded the Glitch into the page

Reply:
Cant seem to see the glitch in the preview though